Include/internal/pycore_frame.h

```diff
@@ -130,7 +136,7 @@ _PyFrame_Initialize(
     frame->stacktop = code->co_nlocalsplus;
     frame->frame_obj = NULL;
     frame->instr_ptr = _PyCode_CODE(code);
-    frame->next_instr_offset = 0;
+    frame->return_offset = 0;
```

Comment:
Do we even need to initialize it here at all? (Ditto below in `_PyFrame_PushTrampolineUnchecked`.)